Tensile/KernelWriterAssembly.py

```diff
@@ -1738,12 +1738,14 @@ def initKernel(self, kernel, tPA, tPB ):
     self.defineSgpr("KernArgAddress", self.rpga)
     assert(self.sgprs["KernArgAddress"] ==  0) # kernarg is passed to kernel as SGPR0
 
-    if kernel["WorkGroupMapping"]>=0 :
-      self.defineSgpr("WorkGroup0", 1)
-      self.defineSgpr("WorkGroup1", 1)
-    else:
-      self.defineSgpr("WorkGroup1", 1)
-      self.defineSgpr("WorkGroup0", 1)
+    #if kernel["WorkGroupMapping"]>=0 :
+    #  self.defineSgpr("WorkGroup0", 1)
+    #  self.defineSgpr("WorkGroup1", 1)
+    #else:
+    #  self.defineSgpr("WorkGroup1", 1)
+    #  self.defineSgpr("WorkGroup0", 1)
+    self.defineSgpr("WorkGroup0", 1)
+    self.defineSgpr("WorkGroup1", 1)
```

Comment:
Are these now needed for WGM==0 case?

Reply:
> Are these now needed for WGM==0 case?

I did not change anything for WGM==0 for this code.
We do not need to swap definition of WorkGroup1 and WorkGroup0 here for negative WGM.
It makes logic more complicated and negative WGM works without swapping them here.

I do not know what is the actual difference between WGM=0 and 1.
As long as I checked, asm code is exactly same (except for kernel name).